core/framework/test/suite/dsls/xvm/page/05.model/component/externalclasses/index.test.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2021-present, hapjs.org. All rights reserved.
```

Comment:
/*
 * Copyright (c) 2021-present, the hapjs-platform Project Contributors
 * SPDX-License-Identifier: Apache-2.0
 */